### DISCUSSION TITLE
Metallic look on android; no textures iOS

### DISCUSSION THREAD

**Discussion by JSadones** [google/model-viewer#3055] [Q&A]
Hi,

Does anybody know why the following is not working?

OnePlus 8t:
https://i.imgur.com/jmEHvAv.jpg

Here the gifts look metallic shiny.

On a colleague iPhone device, he doesn't get textures at all:
https://i.imgur.com/DaHzt5N.jpg

On web browser this all looks normal:
https://i.imgur.com/bgVW84F.png

Setup in blender should be standard as well.

**Comment by elalish** [comment]
Can you just paste your images here instead of sharing imgur links? The last one has  already broken, so it's hard to tell what the problem is. Can you share a repro URL? What version of MV are you using?
